examples/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ function(add_test_executable TARGET_NAME)
 endfunction()
 
 add_test_executable(test_simple_race)
+add_test_executable(demo_interface)
```

Comment:
The name of this file is not intuitive. Change a better name about what is done.